### ISSUE TITLE
Set up GitHub pages

### ISSUE THREAD

**Issue by raamana** [raamana/neuropredict#12]
Readthedocs.org is unreliable. 

Need to set up GitHub pages and use the existing sphinx docs to autogenerate GitHub pages.

**Comment by neex-io** [comment]
I am looking for a good (doable, but challenging) first #hacktoberfest issue. I would like to grab this one out.

**Comment by raamana** [comment]
Thank you :), appreciate your help. 

First, Are you familiar with 
 - methods to automatically generate documentation, with sphinx?
 - methods to set up websites containing that documentation - like readthedocs.io and github pages?

Let's try to discuss this here, and make a plan before you dive into code and hack your way.

**Comment by neex-io** [comment]
Sounds great. 
I have never written sphinx documentation. But I generated a few things locally, and I did read a little about it for a project before (https://github.com/codeforgoodconf/free-geek). I had to write a script to fix links issue, but I got a good idea how Sphinx work.

I did use gh-pages a lot, but all via Jekyll.
 
I did google for help and found this: https://daler.github.io/sphinxdoc-test/includeme.html and this http://nikhilism.com/post/2012/automatic-github-pages-generation-from/. I thought I could try to set it up.
